test/module/irohad/pending_txs_storage/pending_txs_storage_test.cpp

```diff
@@ -130,6 +129,127 @@ TEST_F(PendingTxsStorageFixture, InsertionTest) {
   }
 }
 
+/**
+ * Timestamp in PaginationMeta works in PendingTxsStorage
+ * @given two batches of two transactions and storage
+ * @when storage receives updated mst state with the batch
+ * @then list of pending transactions beetwen two timestamps can be obtained
+ */
+TEST_F(PendingTxsStorageFixture, TxPaginationTestFirsTimeLastTimeSpecified) {
+  auto state = emptyState();
+  auto first_time = getUniqueTime();
+  auto transactions = twoTransactionsBatch();
+  auto last_time = getUniqueTime();
```

Comment:
Don't use time in tests. It leads to undetermined states. Just numbers as timestamps.